src/client/index.ts

```diff
@@ -49,13 +59,14 @@ export class SocialInboxClient {
 
     const finalContentType = contentType ?? TYPE_TEXT
 
-    // TODO: Signing
     const response = await this.fetch(url, {
       method,
       headers: {
         'Content-Type': finalContentType
       },
-      body
+      body,
+      publicKeyId: this.keypair.publicKeyId,
```

Comment:
This field does not exist, it should be passed in as an option. Surprised the lint step missed this.